src/routers/router.ts

```diff
@@ -11,26 +10,19 @@ import validateApiKey from '../middleware/validateApiKey';
 import validateAdminApiKey from '../middleware/validateAdminApiKey';
 import attachRedisClient from '../middleware/attachRedisClient';
 import analytics from '../middleware/analytics';
+import { rateLimiter } from '../middleware/rateLimiter';
 
 const router = express.Router();
 
-const FIFTEEN_MINUTES = 15 * 60 * 1000;
-const REQUESTS_PER_MS = 20;
-const createDeveloperLimiter: Express.MiddleWare = rateLimit({
-  windowMs: FIFTEEN_MINUTES,
-  max: REQUESTS_PER_MS,
-});
-
-// Google Analytics
-router.use(analytics);
+router.use(analytics, rateLimiter);
 
 router.get('/words', validateApiKey, attachRedisClient, getWords);
 router.get('/words/:id', validateApiKey, validId, attachRedisClient, getWord);
 router.get('/examples', validateApiKey, attachRedisClient, getExamples);
 router.get('/examples/:id', validateApiKey, validId, attachRedisClient, getExample);
 
-router.post('/developers', createDeveloperLimiter, validateDeveloperBody, postDeveloper);
+router.post('/developers', validateDeveloperBody, postDeveloper);
 
-router.get('/stats', validateAdminApiKey, attachRedisClient, getStats);
+router.get('/stats', rateLimiter, validateAdminApiKey, attachRedisClient, getStats);
```

Comment:
why add `rateLimit` specifically to this route? it's already applied to this router

Reply:
A little error